src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml

```diff
@@ -15,10 +15,11 @@
             var path = paths[i];
             var url = Orchard.AssetUrl(path);
             var text = !string.IsNullOrEmpty(mediaTexts.ElementAtOrDefault(i)) ? mediaTexts[i] : path;
+            var alt = !string.IsNullOrEmpty(mediaTexts.ElementAtOrDefault(i)) ? mediaTexts[i] : string.Empty;
```

Comment:
Can we move the line 18 in the else block "line 25"?

Reply:
It would need to be in the if block on line 22.

Reply:
@MikeAlhayek how does https://github.com/OrchardCMS/OrchardCore/pull/16859/commits/63928d7cb9a87e2025d4b081d5ac83feda2d1ea6  look?

Reply:
I could move the `var text` assignment (line 17) to inside the else block now that it isn't used in the if block.

Reply:
@DrewBrasher 
Yes also please move line 17 "var text" to the else block. 

Additionally, please change paths.Any() on line 9 to paths.length > 0

Reply:
I made those changes